setup.cfg

```diff
@@ -85,7 +83,10 @@ install_requires =
     packageurl-python>=0.11.1
     packaging>=23.1
     pathspec>=0.11.2
+
+    #??
```

Comment:
??

Reply:
I doubt that we're using `Pillow` in FederatedCode. Actually, there are a bunch of dependencies that are not being used. We need to clean up the unused dependencies, this is being tracked in a separate issue here https://github.com/aboutcode-org/federatedcode/issues/43